scripts/apidoc/utils.ts

```diff
@@ -146,3 +146,13 @@ export function joinTagParts(parts?: CommentDisplayPart[]): string | undefined {
 export function isDeprecated(signature: SignatureReflection): boolean {
   return extractTagContent('@deprecated', signature).length > 0;
 }
+/**
+ * Extracts the "since"
```

Comment:
Not sure about this description.

```suggestion
 * Extracts the "since" tag from the provided signature.
```